packages/desktop-client/src/components/reports/CategorySelector.tsx

```diff
@@ -132,6 +133,7 @@ export function CategorySelector({
           paddingRight: 10,
           flexGrow: 1,
           overflowY: 'auto',
+          scrollbarColor: theme.tableBackground + ' ' + theme.pageBackground,
```

Comment:
The app uses a set of scrollbar styles e.g. `styles.lightScrollbar` or `styles.darkScrollbar`. We can try applying either of those styles here to be consistent with other parts of the app.